docs/blog/2020-06-16-Gatsby-slideshow-with-Posenet/index.md

```diff
@@ -384,7 +376,6 @@ class PoseNet extends Component {
         outputStride,
       })
       poses.push(pose)
-      break
```

Comment:
code check says `break has no target` here. 
it looks not right at this place, but i am not sure

Reply:
This "break" can be removed.